packages/neuron-wallet/src/utils/export-history.ts

```diff
@@ -0,0 +1,149 @@
+import fs from 'fs'
+import sqlite3 from 'sqlite3'
+import { get as getDescription } from 'database/leveldb/transaction-description'
+import i18n from 'locales/i18n'
+import shannonToCKB from 'utils/shannonToCKB'
+
+
+interface ExportHistoryParms {
+  walletID: string
+  dbPath: string
+  lockHashList?: string[]
+  filePath: string
+}
+const exportHistory = ({
```

Comment:
Since it involves quite a number of logics in different parts, I would suggest having some unit tests or integration tests to cover them.

Reply:
Sure, but this feature is urgent and the test cases of `TransactionService.getAll` are absent, so I've decided to add them a few later.